src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultArray = {};
+  const splitString = sourceString.split(';');
+
+  splitString.map((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      resultArray[splitElement[0]] = splitElement[1];
```

Comment:
There's a potential issue if 'splitElement' doesn't have two items after the split operation. This could happen if there's a CSS declaration without a property or value. It's a good practice to add a check to ensure 'splitElement' has the expected number of items before assigning them to 'resultArray'.